client/src/tests/modules/hoc/index.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { shallow } from "enzyme";
+import renderCards from "../../../modules/HOC";
+
+describe("<Test HOC Component />", () => {
+  it("should mount without crashing", () => {
+    const addFavorites = jest.fn(3);
+    const data = {
+      teams: [
+        {
+          private: 'private team',
+          favorite: 'red',
+          progress: 25
+        },
+        {
+          private: 'private team',
+          favorite: 'red',
+          progress: 65
+        },
+        {
+          private: 'private team',
+          favorite: 'red',
+          progress: 244
+        }
+      ]
+    };
+    const props = {
+      item: {},
+      favorite: "fargo",
+      lock: "andela",
+      toolTip: "taps",
+      progressBar: data.teams,
+      addFavorites,
+      isFavorited: true
+    };
+
```

Comment:
Good job and making the test coverage 100 % is a nice one !!
Looking at the way you have your test mock data in the test, what do you think about separating it into a mock file ? do you think it will make your code neater and shorter?

Reply:
I have implemented your feedback.